docs/7.x/changelog.md

```diff
@@ -20,6 +20,8 @@ more current versions of Kubernetes. Regular branches of Gravity are supported
 with security and bug fixes until first release of the subsequent Gravity
 branch is published.
 
+_* Unless extended through customer agreement._
```

Comment:
Why this addendum instead of adjusting the earlier line to 3 years?

> Gravity LTS versions are supported with security and bug fixes for two years.

Is there a difference between LTS support for one customer and across-the-board LTS support?  Either way it seems like we need to do the same proactive security backports.

It seems like the edge case would be in year 3 where an OSS customer (or paying customer with without an agreement) hits a an LTS bug that the paying sponsor doesn't care about, and we don't backport it?

Reply:
> Is there a difference between LTS support for one customer and across-the-board LTS support?

Yes, it might make other customers upgrade faster :) Increasing this to 3 years across the board will significantly increase our support burden.